docs/reference/actions.md

```diff
@@ -1 +1,25 @@
-Actions are not yet implemented in CIM Database Cloud.
+Functions can return a list of "Actions" that should be performed in CIM Database Cloud. Not all Events support the same actions, so check the supported actions in the [Events documentation](events.md). For example Events that are triggered **after** the release of an object don't support AbortAndShowError, because the release can't be aborted anymore, however the "release check" events do support it.
+
+### Return an Action:
+
+Actions need to be packaged in a WorkloadResponse, however csfunctions will handle that for you if your Function returns an Action or a list of Actions:
```

Comment:
If the WorkloadResponse is not needed in normal operation I would not mention it here but somewhere separately. Otherwise it is confusing here

Reply:
[3783930](https://github.com/cslab/functions-sdk-python/pull/13/commits/37839304e800894b20c683cff638e9904768b0bc)